src/CliMenuBuilder.php

```diff
@@ -136,13 +136,9 @@ public function addLineBreak(string $breakChar = ' ', int $lines = 1) : self
         return $this;
     }
 
-    public function addAsciiArt(string $art, string $position = AsciiArtItem::POSITION_CENTER) : self
+    public function addAsciiArt(string $art, string $position = AsciiArtItem::POSITION_CENTER, string $alt) : self
     {
-        $asciiArtItem = new AsciiArtItem($art, $position);
-
-        if ($asciiArtItem->getArtLength() <= $this->getMenuStyle()->getContentWidth()) {
-            $this->addMenuItem($asciiArtItem);
```

Comment:
Maybe it would be cleaner to just add a static item here instead? Then we wouldn't have to modify `AsciiArtItem`

Reply:
Then it wouldn't work if you first create the AsciiArtItem and then add it using addMenuItem().